nunit-project-editor.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-project-editor</id>
+    <version>1.0</version>
```

Comment:
It can and our nuget packages do. I just need to look up how to do it with choco as opposed to nuget.